business/apps.go

```diff
@@ -219,6 +224,7 @@ func (in *AppService) GetAppDetails(ctx context.Context, criteria AppCriteria) (
 	(*appInstance).Workloads = make([]models.WorkloadItem, len(appDetails.Workloads))
 	for i, wkd := range appDetails.Workloads {
 		(*appInstance).Workloads[i] = models.WorkloadItem{WorkloadName: wkd.Name, IstioSidecar: wkd.IstioSidecar, Labels: wkd.Labels, ServiceAccountNames: wkd.Pods.ServiceAccounts()}
+		(*appInstance).Workloads[i] = models.WorkloadItem{WorkloadName: wkd.Name, IstioSidecar: wkd.IstioSidecar, IstioAmbient: wkd.IstioAmbient, ServiceAccountNames: wkd.Pods.ServiceAccounts()}
```

Comment:
I'm confused here. Is this line duplicated, with the additional istioAmbient in the second line? Shouldn't line 226 be deleted?

Reply:
You are right! Line 226 is deleted now.